src/main/scala/com/microsoft/hyperspace/actions/RefreshAction.scala

```diff
@@ -36,9 +36,9 @@ class RefreshAction(
     dataManager: IndexDataManager)
     extends RefreshActionBase(spark, logManager, dataManager) {
 
-  final override def op(): Unit = {
-    write(spark, df, indexConfig)
-  }
+  override def logEntry: LogEntry = getIndexLogEntry(spark, df, indexConfig, indexDataPath)
+
+  final override def op(): Unit = write(spark, df, indexConfig)
```

Comment:
nit: do we need these changes?

Reply:
It is a single line def and we dont add extra {} and new lines. We have used this style in some other parts of code, s.a. [RefreshActionBase](https://github.com/microsoft/hyperspace/blob/master/src/main/scala/com/microsoft/hyperspace/actions/RefreshActionBase.scala#L81) or [IndexLogEntry](https://github.com/microsoft/hyperspace/blob/master/src/main/scala/com/microsoft/hyperspace/index/IndexLogEntry.scala#L468).